src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/design/finalclass/InputFinalClassClassWithPrivateCtorWithNestedExtendingClass.java

```diff
@@ -10,17 +10,18 @@
 public class InputFinalClassClassWithPrivateCtorWithNestedExtendingClass {
     class A {
         private A() {}
-        private class ExtendA extends A {}
+        private final class ExtendA extends A {}
```

Comment:
Inputs should not be modified.

Reply:
@rnveach that no issue i will remove it. after our new changes that is expected

Reply:
done